interface-definitions/include/firewall/add-dynamic-address-groups.xml.i

```diff
@@ -0,0 +1,49 @@
+<!-- include start from firewall/add-dynamic-address-groups.xml.i -->
+<leafNode name="address-group">
+  <properties>
+    <help>Dynamic address-group</help>
+    <completionHelp>
+      <path>firewall group dynamic-group address-group</path>
+    </completionHelp>
+  </properties>
+</leafNode>
+<leafNode name="timeout-value">
+  <properties>
+    <help>Set timeout</help>
+    <valueHelp>
+      <format>u32:1-255</format>
+      <description>Remains element in group for this time</description>
```

Comment:
This sentence is ungrammatical. Possible alternative: `How long elements remain in the group`.